### PR TITLE
Update branch name highlight

### PR DESCRIPTION
**Pull request by Morganov**
- [x] I provide my work under the [project license](https://github.com/progit/progit2/blob/master/LICENSE.asc).
- [x] I grant such license of my work as is required for the purposes of future print editions to [Ben Straub](https://github.com/ben) and [Scott Chacon](https://github.com/schacon).

## Changes

- The branch name should be `monospaced` if it is not a reference to the word.

### CODE REVIEW

File: book/03-git-branching/sections/branch-management.asc [hunk 1]
```diff
@@ -162,10 +160,9 @@ git branch --all
   remotes/origin/master
 ----
 
-Your local _master_ branch is gone, as it's replaced with the _main_ branch.
-The _main_ branch is also available on the remote.
-But the remote still has a _master_ branch.
-Other collaborators will continue to use the _master_ branch as the base of their work, until you make some further changes.
+Your local `master` branch is gone, as it's replaced with the `main` branch.
+The `main` branch is available on the remote but the `master` still also there.
```
Comment (HonkingGoose) [inline book/03-git-branching/sections/branch-management.asc:164]:
```suggestion
The `main` branch is present on the remote.
However the old `master` branch is still present on the remote.
```

How about something like this?

Comment (Morganov) [reply book/03-git-branching/sections/branch-management.asc]: OK, but comma after the "however". Updated.

**Comment by ben** [comment]
This looks great to me. @HonkingGoose are you good with it?

**Comment by HonkingGoose** [comment]
@ben I'm good with this. 😄 Permission to merge! 🚀

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**